apps/frontend/src/pages/chat/libs/components/buttons-controller/libs/components/suggestions-creation-options/libs/enums/suggestions-creation-button-label.enum.ts

```diff
@@ -0,0 +1,6 @@
+const SuggestionsCreationButtonLabel = {
+	NO: "No, smth else",
```

Comment:
Let's use the full version of word "something"

Reply:
It was like that in design

Reply:
Agree, let's use full version